src/libs/antares/study/parts/common/cluster.cpp

```diff
@@ -20,12 +20,10 @@
 */
 #include "antares/study/parts/common/cluster.h"
 
-#include <yuni/yuni.h>
```

Comment:
Here we remove **Yuni** headers.
Technically we should not because `Yuni::IO::Separator` is still referenced in the current file.
But it would be not much of a work to use `std::filesystem` instead.